data-prepper-plugins/aggregate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/aggregate/AggregateActionInput.java

```diff
@@ -16,4 +18,11 @@ public interface AggregateActionInput {
      * @since 1.3
      */
     GroupState getGroupState();
+
+    /**
+     * @return a map of Identification keys map containing identification keys and
+     *         their values
+     * @since 2.1
+     */
+    Map<Object, Object> getIdentificationKeysMap();
```

Comment:
I think this could just be `getIdentificationKeys()`. No need for redundant map text.

Reply:
I think this naming is a little misleading given that the `AggregateProcessorConfig` also has the `getIdentificationKeys` method, and it is different from what is returned in this `getIdentificationKeys` method. I am ok with leaving as is, but would prefer we differentiate